models/user.js

```diff
@@ -61,17 +67,30 @@ const userSchema = new Schema(
     },
     appLanguage: {
       type: String,
-      enum: APP_LANG_ENUM,
+      enum: {
+        values: APP_LANG_ENUM,
+        message: `App language can be only one of these: ${APP_LANG_ENUM.toString()}`
+      },
       default: APP_LANG_ENUM[0],
       select: false
     },
     status: {
       type: String,
-      enum: STATUS_ENUM,
+      enum: {
+        values: STATUS_ENUM,
+        message: `User status can be only one of these: ${STATUS_ENUM.toString()}`
+      },
       default: STATUS_ENUM[0],
       select: false
     },
-    lastLoginAs: { type: String, enum: [STUDENT, TUTOR, ADMIN], select: false },
+    lastLoginAs: {
+      type: String,
+      enum: {
+        values: LOGIN_ROLE_ENUM,
+        message: `User's last login should be as one of these: ${LOGIN_ROLE_ENUM.toString()}`
```

Comment:
remove 's from user

can be
can be
should be
stick to 'can be'

Reply:
since it is referred to a field you can use both, but since it is omitted for other messages, then it should be omitted here as well
u - uniformity

Reply:
fixed